### ISSUE TITLE
Add Users to Organizations

### ISSUE THREAD

**Issue by luanfonceca** [luanfonceca/speakerfight#285]
This issue depends on #130.

We need to create a field to add Users into a Organization. This field is called `members`.

In the "Invite more people" field, we can only asks for a email address and try to relate that user with this Organization and send a email telling to the user that he was joined. If the users doesnt exists on speakerfight we can just return an error telling that this user needs to sign up before be invited in a organization.

![screen shot 2017-10-03 at 19 05 04](https://user-images.githubusercontent.com/1490875/31151457-e794c744-a86d-11e7-8d16-583f43f9b8b9.png)

**Comment by arturfelipe** [comment]
Hey @luanfonceca can I help with this one?

**Comment by luanfonceca** [comment]
Yeah, sure! If you need some help please come to http://gitter.im/luanfonceca

**Comment by arturfelipe** [comment]
Hey @luanfonceca what do you think about adding members with some membership types, like Owner? That way I believe that queries will be easier to get members, and still keeping the ownership info, and why not enable more than one owner per org. Let me know if you just want to keep the `created_by` field and create a `members` relation.
